nautobot/extras/tests/test_models.py

```diff
@@ -209,22 +209,23 @@ def test_name_ordering_after_weight(self):
 
     def test_name_uniqueness(self):
```

Comment:
Do we still need this test since `unique=True` is guaranteed to work? I guess this wouldn't hurt either

Reply:
We could probably remove it, yeah - I went the lazy route of just changing the test logic to reflect the behavior change, but you're right that it's testing pretty core Django capabilities now.